source/index.html.md

```diff
@@ -538,6 +538,119 @@ ID | The ID of the talent agency to retrieve
+  -H "Authorization: Token token=API_KEY"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
+  {
+    "id": 36,
+    "title": "Blog post title"
+  },
+  {
+    "id": 35,
+    "title": "Blog post title"
+  }
+]
+
+```
+
+This endpoint retrieves all blog posts available with the access rights of the API key.
+
+### Pagination
+
+This API endpoint is paginated, following the proposed [RFC-8288](https://tools.ietf.org/html/rfc8288) standard for Web linking.
+
+The pagination information is included in response HTTP headers, namely `Total` (total entries), `Per-Page` (entries per page) and `Link` (containing links to corresponding pages). The current page can be set using the `page`-parameter.
+
+### HTTP Request
+
+`GET https://www.filmmakers.eu/api/v1/blog_posts`
+
+### Query Parameters
+
+Parameter | Default | Description
+--------- | ------- | -----------
+page | 1 | Page to display - see "Pagination" above
+
+### Response fields
+
+Field | Type | Description
+--------- | ------- | -----------
+id | number | Unique ID of the blog post
+title | string | Title of the blog post
+
+## Get a specific blog post
+
+```shell
+curl "https://www.filmmakers.eu/api/v1/blog_posts/123" \
+  -H "Authorization: Token token=API_KEY"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "id": 123,
+  "title": "Blog post title",
+  "note": "Blog post note",
+  "publication_date": "2019-09-02T12:44:19.776+02:00",
+  "source": null,
+  "body": "Blog post body",
+  "body_html": "<div>Blog post body<figure class=\"attachment attachment--preview attachment--jpg\"><div class=\"image-wrapper\"><img src=\"https://www.filmmakers.eu/rails/active_storage/representations/redirect/abc/image.jpg\"><div class=\"image-caption\"><span title=\"© Acme inc\">© Acme inc</span></div></div></figure></div>",
+  "images": [
+    {
+      "url": "https://xxx.cloudfront.net/",
+      "copyright": "Acme inc"
+    },
+    {
+      "url": "https://xxx.cloudfront.net/",
+      "copyright": "Abc inc"
+    }
+  ]
+}
+
+```
+
+This endpoint retrieves a specific blog post.
+
+### HTTP Request
+
+`GET https://www.filmmakers.eu/api/v1/blog_posts/<ID>`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+ID | The ID of the blog post to retrieve
+
+### Response fields
+
+See example response to the right for an overview of included fields
+
+Field | Type | Description
+--------- | ------- | -----------
+title | string | title of the blog post
+note | string | optional comment on the post
+body | string | plain text representation of the blog post body with line breaks, **but no HTML markup**
+body_html | string |  blog post body with **HTML markup**, includes images with copyright information. Following example CSS can be used to show copyright over the image at the top left corner: `.image-wrapper { position: relative; } .image-caption { position: absolute; top: 0; left: 0; z-index: 1; }`
```

Comment:
@sunil-filmmakers please document the possible HTML tags that are allowed here. I received this question from an api user - they want to customize some styles (e.g. headings) and therefore want to know all possible cases.